development/Dockerfile

```diff
@@ -19,14 +19,15 @@ FROM ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER}-py${PYTHON_VER}
 ARG NAUTOBOT_ROOT=/opt/nautobot
 
 ENV prometheus_multiproc_dir=/prom_cache
-ENV NAUTOBOT_ROOT ${NAUTOBOT_ROOT}
+ENV NAUTOBOT_ROOT=${NAUTOBOT_ROOT}
+ENV INVOKE_NAUTOBOT_DEVICE_LIFECYCLE_MGMT_LOCAL=true
```

Comment:
This seems a little strange to be inside the dockerfile, do we need this?

Reply:
Yes, with this, when running `invoke` inside the docker container, it uses local by default.

Without this environment variable, it would try to use `docker compose` inside the docker container.

Reply:
I'm using it quite often, adding this environment variable into the container is safe. Can be used in CI in the future as well.